policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/yt_employment_amount/yt_employment_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Yukon provides this employment amount. 
+values:
+  2022-01-01: 1_287
+metadata:
+  unit: currency-CAD
+  label: Yukon Employment Amount
+  reference:
+  - title: Canada Employment Amount Benefit for Yukon
```

Comment:
need to change to actual title of the document + add the yukon income tax act reference

Reply:
in both references we need to specify the paragraph and page where the value is present